src/StructuredLogger/Reflector.cs

```diff
@@ -71,18 +71,6 @@ public static void SetTimestamp(BuildEventArgs args, DateTime timestamp)
             timeStampSetter(args, timestamp);
         }
 
-        private static Action<BuildMessageEventArgs, int> lineNumberSetter = GetFieldSetter<BuildMessageEventArgs, int>("lineNumber");
-        public static void SetLineNumber(BuildMessageEventArgs args, int lineNumber)
-        {
-            lineNumberSetter(args, lineNumber);
-        }
-
-        private static Action<BuildMessageEventArgs, int> columnNumberSetter = GetFieldSetter<BuildMessageEventArgs, int>("columnNumber");
-        public static void SetColumnNumber(BuildMessageEventArgs args, int columnNumber)
-        {
-            columnNumberSetter(args, columnNumber);
-        }
-

```

Comment:
I don't think we need this anymore as `TaskParameterEventArgs2` contains new props representing line and column number.